src/SmartComponents/CompletedTaskDetails/JobResultsDetails/EntryDetails.js

```diff
@@ -32,7 +32,13 @@ const EntryDetails = ({ entry, taskConstantMapper }) => {
             </span>
           ) : null}
           <span className={classname}>
-            {item.type ? `[${item.type}] ${item.context}` : `${item.context}`}
+            {item.type
+              ? `[${item.type}] ${
+                  item.type == 'command' && Array.isArray(item.context)
```

Comment:
You'll want to use a `===` here instead of `==`. See an example below of why. The `==` performs a type conversion before comparing.

```
const a = 100;
const b = '100';

console.log(a == b) // true
```